extensions/spotify-player/src/browseAll.tsx

```diff
@@ -10,10 +10,10 @@ export default function BrowseAll() {
   }
 
   return (
-    <List searchBarPlaceholder="Search Genres..." isLoading={response.isLoading} throttle>
+    <Grid searchBarPlaceholder="Search genres..." isLoading={response.isLoading} throttle>
```

Comment:
Nice! Makes a lot of sense

Reply:
We don't have a rule of having capitalised words in the searchbar, and in other screens it looks the same as far as I remember 🤔

Reply:
Sorry I meant the use of the grid, wasnt trying to make a comment about the casing!